TrackingTools/TrackAssociator/plugins/MuonDetIdAssociator.cc

```diff
@@ -77,7 +79,14 @@ void MuonDetIdAssociator::getValidDetIds(unsigned int subDectorIndex, std::vecto
       for(std::vector<const RPCRoll*>::iterator r = rolls.begin(); r != rolls.end(); ++r)
 	validIds.push_back((*r)->id().rawId());
     }
-  
+
+  // ME0
+  if ( doME0_ ){
+    if (! geometry_->slaveGeometry(ME0DetId()) ) throw cms::Exception("FatalError") << "Cannnot ME0Geometry\n";
```

Comment:
Exception message doesn't make sense and is misspelled. `Cannnot ME0Geometry` ???